tests/k8s/test_client.py

```diff
@@ -96,6 +96,17 @@ def test_watch_list(self, session):
             "GET", _absolute_url("/watch/example"), json=None, timeout=None, stream=True
         )
 
+    @pytest.mark.parametrize("key", (
+            "client_id",
+            "authorization"
+    ))
+    def test_redacts_sensitive_headers(self, key):
```

Comment:
Perhaps this test should include all the relevant headers as test cases, since we're parametrizing the test anyway?

Reply:
Agreed, and done.